tests/spiders/test_poland_rjps.py

```diff
@@ -0,0 +1,224 @@
+<em class="fa fa-file-text-o fa-stack-1x"></em>
+</span>
+</div>
+<div class="flex flex-column justify-content-center">
+<span class="wrap-anywhere">Wnioskuj o świadczenie - platforma Emp@tia</span>
+</div>
+</div>
+</a>
+</div>
+
+						
+						
+				</div>
+    		
+    </div>
+    </div>
+    <div class="col-12 col-lg-5 px-0">
+        <div class="data-aktualizacji text-right w-100">
+        	data aktualizacji:  2018-02-05
+        </div>
+        <div class="zdjecie">
+            <div title="Brak zdjęcia">
+                <div class="row justify-content-center">
+                    <div class="col text-center pt-3">Brak zdjęcia</div>
+                </div>
+                <div class="row justify-content-center">
+                    <div class="col text-center pt-4 pb-5">
+                    	<span class="fa-stack fa-lg">
+                    		<i class="fa fa-camera fa-stack-1x"></i>
+                    		<i class="fa fa-ban fa-stack-2x text-danger"></i>
+                    	</span>
+                    </div>
+                </div>
+            </div>
+            <a aria-label="Pokaż kartę dla jednostki" onClick="przegladarka.pokazKarta(3499,5)">
+                <img alt="Symbol brakującego zdjęcia - jednostka nie dodała żadnej fotografii"/>
+            </a>
+        </div>
+    </div>
+    <div class="col-12 pt-2" title="Szczegóły">
+        <button class="btn btn-wj btn-wj-blue" onclick="przegladarka.pokazKarta(3499,5)">
+            Szczegóły
+        </button>
+    </div>
+</div>
+<hr/>
+<script>
+    lista.initJednostka($('#mapaContainer'), 3499);
+</script>
+
+	
+</div>
+    """
+    # TODO: move raw text into file
+    return HtmlResponse(body=raw_text, encoding='utf-8', url='test.com')
+
+
+class TestQuotesSpider:
+    def test_parse_name(self, normal_place):
+        name = QuotesSpider().parse(normal_place, category='test').get('name')
+        assert name == 'Miejski Ośrodek Pomocy Społecznej w Zabłudowie'
+
+    def test_parse_address(self, normal_place):
+        address = QuotesSpider().parse(normal_place, category='test').get('address')
+
+        assert address == '16-060 Zabłudów ul. Rynek 8'
+
+    def test_parse_email(self, normal_place):
+        description = QuotesSpider().parse(normal_place, category='test').get('description')
+        assert 'biuro@mops-zabludow.pl' in description
+
+    def test_parse_phone(self, normal_place):
+        description = QuotesSpider().parse(normal_place, category='test').get('description')
+        assert 'tel. 85 7188100' in description
+
+    def test_parse_website(self, normal_place):
+        description = QuotesSpider().parse(normal_place, category='test').get('description')
+        assert 'http://bip.mops.um.zabludow.wrotapodlasia.pl' in description
+
+    def test_parse_update_date(self, normal_place):
+        description = QuotesSpider().parse(normal_place, category='test').get('description')
+        assert 'data aktualizacji:  2018-02-05' in description
```

Comment:
# TODO 

it's better to format the update date as `data updated: 2018-02-05`

Reply:
https://github.com/littlepea/safeway-data/pull/18/commits/fd705666c5da0a19151a26bd5263698b63437353